sktime/utils/_testing/panel.py

```diff
@@ -105,6 +105,17 @@ def make_regression_problem(
     return X, y
 
 
+def make_clustering_problem(
+    n_instances=20, series_size=20, return_numpy=True, random_state=None
+):
+    rng = check_random_state(random_state)
+    X = [rng.randn(n_instances, series_size)]
```

Comment:
Why is `X` a list here? Shouldn't we use `_make_panel_X(**kwargs)` here?

Reply:
The signature also seems wrong, isn't it: `random.randint(low, high=None, size=None, ...)`?